src/test/controller/UserController.test.ts

```diff
@@ -384,19 +400,75 @@ describe('Unit tests for UserController', function () {
    * Compare actual user against expected user
    * */
   function compareUsers(actualUser: User, expectedUser: User) {
-    const expectedObject: any = { ...expectedUser };
+    const expectedObject: any = {
```

Comment:
Do you get errors when you type `expectedObject` as a User?

```suggestion
    const expectedObject: User = {
```

Reply:
Yes, it throws error when deleting operation.

Reply:
Oh I see, yeah it's fine to keep it as `any` then!